### PR TITLE
[Keymap] rjhilgefort's Corne KBD

### PR DESCRIPTION
**Pull request by rjhilgefort**
## Description

Adding my Corne KBD Layout! Thanks in advance for giving this a look over!

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+```
+
+## Building And Flashing
+
+Note, you'll see that most `crkbd`'s recommend a target of `avrdude` because most kits come with a Pro Micro. I put a Elite-C on mine and thus the target is `dfu`. You can see this in the [rules.mk](./rules.mk) as the `BOOTLOADER` configuration.
+
+#### CLI
+
+```
+sudo make crkbd:rjhilgefort:flash
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/rjhilgefort/readme.md:110]: In general we recommend against the use of sudo - set up the [udev rules](https://docs.qmk.fm/#/faq_build?id=linux-udev-rules) if flashing is not working for you.

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# [🐦 @rjhilgefort](https://twitter.com/rjhilgefort)'s Corne KBD (CRKBD) Layout
+
+My Corne KBD layout is inspired by [my Planck Layout](../../../planck/keymaps/rjhilgefort/readme.md), but with some additional features. Namely, this layout has Linux support.
+
+![](https://imgur.com/wp9ksTI.jpg)
+![](https://imgur.com/fnyiCnE.jpg)
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]:
```suggestion
![](https://imgur.com/wp9ksTIl.jpg)
![](https://imgur.com/fnyiCnEl.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md (hunk 1 above)
Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:110]: got it, thank you for the tip!

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md (hunk 2 above)
Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]: Does the `l` fetch a different image? I'm unfamiliar with what this will do with `imgur`

Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]: Ah, looks like it fetches a smaller image. Sounds good to me!

Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md]: Hmm, after I made this change, now the readme looks like this:

image

I think I prefer the larger image if that's okay.

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+# [🐦 @rjhilgefort](https://twitter.com/rjhilgefort)'s Corne KBD (CRKBD) Layout
+
+My Corne KBD layout is inspired by [my Planck Layout](../../../planck/keymaps/rjhilgefort/readme.md), but with some additional features. Namely, this layout has Linux support.
+
+![](https://imgur.com/wp9ksTIl.jpg)
+![](https://imgur.com/fnyiCnEl.jpg)
```
Comment (rjhilgefort) [inline keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]:
```suggestion
![](https://imgur.com/wp9ksTI.jpg)
![](https://imgur.com/fnyiCnE.jpg)
```

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md (hunk 2 above)
Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]: Here's what it looks like with the larger image:

image

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md (hunk 1 above)
Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:110]:
```suggestion
make crkbd:rjhilgefort:flash
```

**Comment by rjhilgefort** [comment]
I think I've commented back on comments, thanks for the review!

File: keyboards/crkbd/keymaps/rjhilgefort/readme.md (hunk 2 above)
Comment (fauxpark) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]: Those images are unnecessarily large, though.

Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]: Does it hurt anything? They’re hosted on imgur so it’s not like it bloats the repo or anything. It’s a keymap for me and I like the large image in the readme? If others don’t like it they’ll skip past my keymap when exploring. I agree they’re larger than they need to be for the space though and wipe down when they load. Do you know how I can fetch a smaller image but one that still fills the space like the oversized one does? Similar to `l`, but a bit bigger?

Comment (fauxpark) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]: No, it doesn't bloat the repo, but for anyone viewing the image, the filesize is unnecessarily large. The original is about 3MB whereas the thumbnailed version is only 74kB.

Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md:6]:
```suggestion
![](https://imgur.com/wp9ksTIh.jpg)
![](https://imgur.com/fnyiCnEh.jpg)
```

Comment (rjhilgefort) [reply keyboards/crkbd/keymaps/rjhilgefort/readme.md]: Awesome, that's perfect! Thanks for the help with this!

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**